src/contracts/markets/Abstract4626MarketWrapper.sol

```diff
@@ -236,7 +236,6 @@ contract Abstract4626MarketWrapper is Initializable, Ownable {
      * @param amount Amount to transfer. 0 to transfer all balance.
      */
     function transferTokens(address token, address to, uint256 amount) external onlyOwner {
-        require(token != asset && token != market, "Cannot transfer asset or market token");
```

Comment:
Don't you think we should keep this check `token != market`? Because this means that the owner can withdraw some market shares no? 
On the other hand, the owner can upgrade the proxy, so if the owner want to be rogue, this check alone will not be enough.

Reply:
that's a good point. The market tokens will sit in the wrapper so we don't want to owner claiming those.

Reply:
@naddison36 Done in this commit: [ac933a6](https://github.com/OriginProtocol/arm-oeth/pull/167/commits/ac933a694a492fa9355904771f8cd57facbc8783)